### PR TITLE
feat: emulate production left space of hosted app

### PR DESCRIPTION
**Pull request by MitanOmar**

### CODE REVIEW

File: tests/dummy/app/styles/app.scss [hunk 1]
```diff
@@ -6,3 +6,45 @@
...
+ul.uk-navbar-nav > li.uk-active > a {
+  color: white;
+  font-weight: bolder;
+  font-size: larger;
+  margin-left: 20px;
+}
+
+.uk-navbar-nav {
+  margin-right: 20px;
+}
```
Comment (czosel) [inline tests/dummy/app/styles/app.scss:50]: Are those really needed? I don't see the class being used anywhere.

Comment (MitanOmar) [reply tests/dummy/app/styles/app.scss:50]: it's used in the top navbar, it was giving it a better UI

**Comment by MitanOmar** [comment]
Screenshot

image

File: tests/dummy/app/styles/app.scss (hunk 1 above)
Comment (czosel) [reply tests/dummy/app/styles/app.scss]: Ah, I see :+1:

**Pull request merged**